pkg/controller/applicationlayer/applicationlayer_controller.go

```diff
@@ -450,6 +467,30 @@ func getDefaultCoreRuleset(ctx context.Context) (*corev1.ConfigMap, error) {
 	ruleset, err := embed.AsConfigMap(
 		applicationlayer.ModSecurityRulesetConfigMapName,
 		common.OperatorNamespace(),
+		embed.FS,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return ruleset, nil
+}
+
+func (r *ReconcileApplicationLayer) getCorazaRuleSet(ctx context.Context) (*corev1.ConfigMap, error) {
+	ruleset := new(corev1.ConfigMap)
+
+	ruleset, err := getOSWAPCoreRuleSet(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return ruleset, nil
+}
+
+func getOSWAPCoreRuleSet(ctx context.Context) (*corev1.ConfigMap, error) {
+	ruleset, err := embed.AsConfigMap(
+		applicationlayer.DefaultCoreRuleset,
+		common.OperatorNamespace(),
+		coreruleset.FS,
```

Comment:
The naming is confusing here (`getOSWAPCoreRuleSet()` and `getCorazaRuleSet()`). `coreruleset.FS` refers to https://github.com/corazawaf/coraza-coreruleset/tree/main/rules, which I would refer to as the `coraza-ruleset`. Within this `coraza-ruleset`, there is the `@owasp_crs` folder which contains the `coreruleset` taken from https://github.com/coreruleset/coreruleset/tree/main/rules. 

I think the `default-coreruleset` config-map should contain the content of `@owasp_crs`. The rest of `coraza-ruleset` (as I call it :sweat_smile:) is basically what we have in `modsecurity-ruleset` and `tigera.conf`...

Reply:
there was an agreement to do this in a separate PR -- resolved conversation